packages/simtask/src/utils.ts

```
@@ -11,8 +11,15 @@ export function parseArgs(defaults: Record<string, string>) {
   const a = process.argv.slice(2);
   for (let i = 0; i < a.length; i++) {
     const k = a[i];
-    if (!k.startsWith("--")) continue;
-    const v = a[i + 1] && !a[i + 1].startsWith("--") ? a[++i] : "true";
+    if (typeof k !== "string" || !k.startsWith("--")) continue;
+    const next = a[i + 1];
+    let v: string;
```

ESLint

[error] Unexpected let, use const instead.

(functional/no-let)